Sources/Examples/Simple/main.swift

```diff
@@ -30,7 +30,7 @@ func client() throws {
 
     let method = (i < steps - 1) ? "/hello" : "/quit"
     print("calling " + method)
-    let call = c.makeCall(method)
+    let call = try! c.makeCall(method)
```

Comment:
How about just `try`?

Reply:
Updated - was going off of the other `try!` below. Fixed that one too